AlgebraInLean/Chapter01/Sheet05.lean

```diff
@@ -6,13 +6,14 @@ namespace AlgebraInLean
 To represent Cₙ, the group of rotational symmetries of a regular n-gon, we will use a type already
 in Mathlib called `Fin`. `Fin n` is the type of all natural numbers less than n. Elements are
 represented as a pair of a natural number and a proof that it is less than n. A value `k : Fin n`
-represents a rotation by 2kπ/n radians. Like the integers, this group is also abelian.
+represents a rotation by 2kπ/n radians. Like the integers, this group is also abelian. [NeZero n] is
```

Comment:
Should we use degrees here too?

Reply:
I had it in degrees, but the expression felt too complicated. Maybe "2πk/n radians" is clearer?